### DISCUSSION TITLE
List of URLS needed for the action

### DISCUSSION THREAD

**Discussion by audunsolemdal** [actions/cache#983] [Q&A]
I'm running self-hosted github runners in a network which locks down outbound traffic by default

I've found that actions/cache seems to do its caching towards azure blob storage, but the URLs seem to vary per repo.

I would like a list of all URLs so i can avoid this issue from the workflows:

```
Run actions/cache@v3
Warning: Failed to restore: request to *** failed, reason: Client network socket disconnected before secure TLS connection was established
```

Currently I've found:

```
        "ki6cacprodeus1file5.blob.core.windows.net",
        "95s5acprodeus1file6.blob.core.windows.net",
        "gel7acprodeus1file0.blob.core.windows.net",
```

from Google searches I find very few references to these ones, but since there are results they should be multi-tenants storage accounts.

**Comment by Phantsure** [comment]
The URL is not fixed you can use `*` (asterisk synmbol)
